tests/Aspire.Hosting.Tests/Utils/LoggerNotificationExtensions.cs

```diff
@@ -1,13 +1,11 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using IResource = Aspire.Hosting.ApplicationModel.IResource;
-using ResourceLoggerService = Aspire.Hosting.ApplicationModel.ResourceLoggerService;
-using ResourceNotificationService = Aspire.Hosting.ApplicationModel.ResourceNotificationService;
-
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
+using ResourceLoggerService = Aspire.Hosting.ApplicationModel.ResourceLoggerService;
+using ResourceNotificationService = Aspire.Hosting.ApplicationModel.ResourceNotificationService;
```

Comment:
Why are these `using X = ...` necessary?

Reply:
I think it's because at some point there was a conflict between an included file and a reference one. Not necessary anymore so removing.